lib/SPIRV/SPIRVToLLVMDbgTran.cpp

```diff
@@ -536,6 +536,11 @@ SPIRVToLLVMDbgTran::transTypeComposite(const SPIRVExtInst *DebugInst) {
           transTypeMember(SPVMemberInst, DebugInst, cast<DIScope>(CT));
       EltTys.push_back(MemberMD);
       DebugInstCache[SPVMemberInst] = MemberMD;
+    } else if (MemberInst->getExtOp() == SPIRVDebug::TypeInheritance) {
```

Comment:
Should this "else if" code also be guarded by a isNonSemanticDebugInfo() ?

Reply:
I don't think so, but I'm not really sure. Maybe @MrSidims can justify this as I just followed his change above for TypeMember

Reply:
We do not need to check here. The check will be found inside transTypeInheritance when we are processing the operands.